easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.6.5-GCC-4.8.3.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '1.6.5'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '4.8.3'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+
+patches = ['OpenMPI-1.6.5-vt_cupti_events.patch']
+
+configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple --with-openib '
+configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
+configopts += '--disable-dlopen '  # statically link component, don't do dynamic loading
+configopts += '--with-hwloc=$EBROOTHWLOC '
+
+dependencies = [('hwloc', '1.8.1')]
+
+# needed for --with-openib
+if OS_NAME in ['debian', 'ubuntu']:
+    osdependencies = ['libibverbs-dev']
+else:
+    # OK for OS_NAME == redhat, fedora, RHEL, SL, centos
+    osdependencies = ['libibverbs-devel']
```

Comment:
OK, this is old. You can pass a tuple to osdependencies now for different distro's. Replace by:
```python
osdependencies = [('libibverbs-dev','libibverbs-devel')]
```